Public/Sdk/Public/Managed/Frameworks/netstandard2/module.config.dsc

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 module({
-    name: "Sdk.Managed.Frameworks.NetStandard2.0",
+    name: "Sdk.Managed.Frameworks.NetStandard2",

```

Comment:
May I ask why this change? The TFM is still NetStandard2.0. I was hoping to keep the module name to match the TFM

Reply:
I thought it doesn't really matter what minor we use, hence I decided to call it `NetCoreApp3` and `NetStandard2`. If we wan't to keep more flavors it makes sense to be specific, I'll revert.

Reply:
We should match what nguet does since they are the 'owners' of these names and the semantics in co-owership with .net team.

---
In reply to: [283755776](https://github.com/microsoft/BuildXL/pull/298#discussion_r283755776) [](ancestors = 283755776)